build_scripts/windows/README.md

```diff
@@ -0,0 +1,27 @@
+Building the Windows MSI Installer
```

Comment:
shall we put this build_scripts under scripts folder? 
kind of confuse as those scripts are also build scripts

Reply:
move windows folder to scripts